base/error.go

```diff
@@ -31,12 +32,15 @@ type SGError struct {
 
 var (
 	ErrAlreadyImported = &SGError{alreadyImported}
+	ErrImportCancelled = &SGError{importCancelled}
 )
 
 func (e SGError) Error() string {
 	switch e.code {
 	case alreadyImported:
 		return "Document already imported"
+	case importCancelled:
```

Comment:
Can you add a brief comment to the returned string on the implications of an import being cancelled?  Eg, "Import cancelled.  Sync Gateway will not reflect this change"  And maybe some verbose comments on why that can happen in the comments, eg "The import was cancelled because it retried too many times and then gave up.  This means the change will not get reflected in Sync Gateway, and so it may end up getting clobbered by later Sync Gateway updates"